dsl/src/main/scala/com.crobox.clickhouse/dsl/misc/RandomStringGenerator.scala

```diff
@@ -0,0 +1,20 @@
+package com.crobox.clickhouse.dsl.misc
+
+import java.security.SecureRandom
+
+import scala.annotation.tailrec
+
+object RandomStringGenerator {
+  private lazy val numberGenerator: SecureRandom = new SecureRandom
+
+  def random(length: Int = 6): String =
+    padLeft(Integer.toString(numberGenerator.nextInt(Int.MaxValue - 1) + 1, 36), length)
```

Comment:
So it's a single random char with 5 '0' by default?

Reply:
Maybe want to reuse the `Random.alphanumeric.take(length).mkString("")` instead?

Reply:
This code is taken from CUID ;-). I'll write some tests for it